tools/kebab-migration.toit

```diff
@@ -0,0 +1,168 @@
+import cli
+import encoding.json
+import host.pipe
+import host.os
+import host.file
+
+main args:
+  cmd := cli.Command "root"
+      --long_help="""
+        Migrates a project from snake-case to kebab-case.
+        """
+
+  code_command := cli.Command "code"
+      --long_help="""
+        Migrates the given source files from snake-case ("foo_bar") to
+        kebab-case ("foo-bar").
+
+        By default uses the Toit SDK that is available through `jag`.
```

Comment:
Do we want to ship this in the SDK, so this becomes more convenient for existing users somehow?

Reply:
Not sure.
I was thinking of just sending our users to a precompiled executable.

Reply:
Let's ship it in sdk/tools for a little while.